migrations/20170622165827-ports-migration.js

```diff
@@ -0,0 +1,130 @@
+module.exports = {
```

Comment:
Yes, exactly. And the migrations are applied when the service starts, and which have run are tracked in a table in the database (managed by a framework called Umzug.